devel/grpc/files/patch-Makefile.diff

```diff
@@ -0,0 +1,100 @@
+--- Makefile.orig	2018-03-01 11:55:17.000000000 -0500
++++ Makefile	2018-03-01 13:32:47.000000000 -0500
+@@ -442,6 +442,7 @@
+ HOST_CXXFLAGS = $(CXXFLAGS)
+ HOST_LDFLAGS = $(LDFLAGS)
+ HOST_LDLIBS = $(LDLIBS)
++HOST_LDLIBSXX = $(LDLIBSXX)
+ 
+ # These are automatically computed variables.
+ # There shouldn't be any need to change anything from now on.
+@@ -2954,7 +2955,7 @@
+ 	$(E) "[LD]      Linking $@"
+ 	$(Q) mkdir -p `dirname $@`
+ ifeq ($(SYSTEM),Darwin)
+-	$(Q) $(LD) $(LDFLAGS) -L$(LIBDIR)/$(CONFIG) -install_name $(SHARED_PREFIX)gpr$(SHARED_VERSION_CORE).$(SHARED_EXT_CORE) -dynamiclib -o $(LIBDIR)/$(CONFIG)/libgpr$(SHARED_VERSION_CORE).$(SHARED_EXT_CORE) $(LIBGPR_OBJS) $(ZLIB_MERGE_LIBS) $(CARES_MERGE_LIBS) $(LDLIBS)
++	$(Q) $(LD) $(LDFLAGS) -L$(LIBDIR)/$(CONFIG) -install_name $(prefix)/lib/$(SHARED_PREFIX)gpr$(SHARED_VERSION_CORE).$(SHARED_EXT_CORE) -dynamiclib -o $(LIBDIR)/$(CONFIG)/libgpr$(SHARED_VERSION_CORE).$(SHARED_EXT_CORE) $(LIBGPR_OBJS) $(ZLIB_MERGE_LIBS) $(CARES_MERGE_LIBS) $(LDLIBS)
```

Comment:
While this potentially looks ok, it would be a zillion time better to fix things upstream. That said, the same comment applies as above. I'm super confused about why this has to be done manually.

Reply:
I agree this would be best fixed upstream. I planned to submit a pr to the gRPC project but this patch is to get the current release building correctly.